packages/dialect-react-ui/components/CardinalAddress/index.tsx

```diff
@@ -0,0 +1,97 @@
+import { useAddressName } from '@cardinal/namespaces-components';
+import { TwitterIcon } from '../Icon/Twitter';
+import type { Connection } from '@project-serum/anchor';
+import type { PublicKey } from '@solana/web3.js';
+
+const formatTwitterLink = (handle: string | undefined, color: string) => {
+  if (!handle) return <a></a>;
+  return (
+    <a
+      href={`https://twitter.com/${handle}`}
+      style={{ color: color }}
+      target="_blank"
+      rel="noreferrer"
+    >
+      {handle}
+    </a>
+  );
+};
+
+function shortenAddress(address: string, chars = 5): string {
+  return `${address.substring(0, chars)}...${address.substring(
+    address.length - chars
+  )}`;
+}
+
+const formatShortAddress = (address: PublicKey | undefined) => {
+  if (!address) return <></>;
+  return (
+    <a
+      href={`https://explorer.solana.com/address/${address.toString()}`}
+      target="_blank"
+      rel="noopener noreferrer"
+    >
+      {shortenAddress(address.toString())}
+    </a>
+  );
+};
+
+const DisplayAddressNew = ({
+  connection,
+  address,
+  height = "13",
+  width = "300",
+  style,
+  color = "#FFFFFF",
```

Comment:
Let's use pass className and use tailwind styles here, like `className="dt-flex dt-gap-1.5"`, etc

Reply:
changed the styling to use tailwind